### ISSUE TITLE
Documentation on website messed up

### ISSUE THREAD

**Issue by cbravo** [serverless-nextjs/serverless-next.js#1796]
Was just reading the documentation on this page: 

https://www.serverless.com/plugins/serverless-nextjs-plugin

Under the heading "Inputs" it looks as if the code for the github table is spat out as plain text.

**Comment by dphang** [comment]
Yeah, unfortunately that is part of the Serverless website which we don't control, I don't think they support markdown very well. Best to read it from the source, which is GitHub for now.

Anyway the docs do need to be revamped; once the core code is refactored and genericized I'll be able to work on it
